boards/arm/cc3235sf_launchxl/board.cmake

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Use the TI OpenOCD (by default in /usr/local/openocd)
+# See the Zephyr project CC3235SF_LAUNCHXL documentation on
+# flashing prerequisites.
+set(OPENOCD "/usr/local/bin/openocd" CACHE FILEPATH "" FORCE)
```

Comment:
We should look at using the Zephyr SDK for this.

Reply:
@galak I tried OpenOCD in the new Zephyr SDK 0.10.1 today, and found that the wifi driver fails to start on the CC3235SF if flashed using it. I don't see this problem when using TI OpenOCD.

Reply:
If you reset the board after flashing with the Zephyr SDK do things work?

Reply:
Can we try SDK 0.10.2 and see how things work.  Even with the reset issue, I'd prefer we use stock zephyr openocd here.